packages/itwinui-css/backstop/tests/assets/selectTag.js

```diff
@@ -12,10 +12,7 @@ class SelectTag extends HTMLElement {
     const dismissible = this.hasAttribute('dismissible');
 
     const innerHtml = `
-    <span
-      class="iui-select-tag"
-      tabindex="0"
-    >
+    <span class="iui-select-tag">
```

Comment:
> I think focusing the whole tag is the right thing to do from a UX perspective ... (the whole tag should be a button maybe) ...

Yes, it maybe good if when the tag is focused, the enter key triggers deselects the tag. Since it might be confusing for the screen reader user to know which close button is for which tag if the close button is the tab indexable element (not too sure how screen readers/assistive tech works).

Like you said, and also like what the article is hinting at, this would be good for accessibility.

But with the mouse, might be good if only the close icon is clickable, and not the entire tag. Because it might be good to open the dropdown when clicking on a tag.